### ISSUE TITLE
Docker is not implementing correct Selinux context.

### ISSUE THREAD

**Issue by pradeeppandeyy** [docker/docker#31780]
**Description**

**Steps to reproduce the issue:**
1. Install docker docker-engine-1.13.0-1.el7.centos.x86_64 and docker-engine-selinux-1.13.1-1.el7.centos.noarch 
2. start container by below command,
docker run -ti --security-opt label=user:unconfined_u --security-opt label=role:unconfined_r --security-opt label=level:s0:c25,c968 image_name bash
3. Go inside the container and check status of ls -lZ command. 

**Describe the results you received:**
I am starting container with s0:c25,c968  level but docker giving different- different context every time. 

**Describe the results you expected:**
Security context level should be same as I am  providing.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `docker version`:**
 

```
docker version
Client:
 Version:      1.13.0
 API version:  1.25
 Go version:   go1.7.3
 Git commit:   49bf474
 Built:        Tue Jan 17 09:55:28 2017
 OS/Arch:      linux/amd64

Server:
 Version:      1.13.0
 API version:  1.25 (minimum version 1.12)
 Go version:   go1.7.3
 Git commit:   49bf474
 Built:        Tue Jan 17 09:55:28 2017
 OS/Arch:      linux/amd64
 Experimental: false
```

**Output of `docker info`:**

```
 docker info
Containers: 11
 Running: 4
 Paused: 0
 Stopped: 7
Images: 19
Server Version: 1.13.0
Storage Driver: overlay
 Backing Filesystem: extfs
 Supports d_type: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins: 
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 03e5862ec0d8d3b3f750e19fca3ee367e13c090e
runc version: 2f7393a47307a16f8cee44a37b262e8b81021e3e
init version: 949e6fa
Security Options:
 seccomp
  Profile: default
 selinux
Kernel Version: 3.10.0-514.el7.x86_64
Operating System: Red Hat Enterprise Linux Server 7.3 (Maipo)
OSType: linux
Architecture: x86_64
CPUs: 1
Total Memory: 672.7 MiB
Name: master2
ID: 3N3X:H36P:4VX7:YLCP:EF7B:PMP7:IHPB:2PB6:SD42:YKY6:DZJQ:AVP5
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: bridge-nf-call-ip6tables is disabled
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false

```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

I am using RHEL 7.3 installed on Virtual-box.

**Comment by cpuguy83** [comment]
I believe this is the known issue in 1.13.0's version of runc which was fixed in 1.13.1
https://github.com/docker/docker/pull/30649

Can you try with either 1.13.1 or (preferably) 17.03.0? Note that 17.03.0 is just a version scheme change with some critical patches that would have just been 1.13.2 otherwise.

**Comment by pradeeppandeyy** [comment]
Hello cpuguy83,

Thanks working fine on  1.13.1.
